packages/contracts/README.md

```diff
@@ -1,51 +1,61 @@
 # JWT Contracts
 
+This directory contains verifier contracts for handling JSON Web Tokens(JWTs) proofs. 
 
-Build
+## Contracts Overview
 
-```
-$ yarn build
-```
+### JWTRegistry
 
-Unit tests
+- **Purpose**: The `JWTRegistry` contract is used to store and manage JWTs. It keeps track of all issued tokens and their statuses.
+- **Usage**: In another project, you can use this contract to verify if a JWT is valid and to check its status (active, revoked, etc.).
 
-```
-$ yarn test:unit
-```
+### JWTVerifier
 
-Integration tests
+- **Purpose**: The `JWTVerifier` contract is responsible for verifying the authenticity of JWTs. It checks the proof and ensures the token is valid.
+- **Usage**: Use this contract in your project to verify JWTs proof before granting access to protected resources.
 
-```
-$ yarn test:integration
-```
+## How to Use in Another Project
+
+1. **Installation**: First, install the package using npm. Run the following command in your project directory:
+
+   ```
+   npm install @zk-jwt/zk-jwt-contracts
```

Comment:
@wshino This should be `yarn add` instead of `npm install` as we use yarn as the package manager.

Reply:
Agree. I updated.